pkg/core/dao/dao.go

```diff
@@ -435,31 +435,38 @@ func (dao *Simple) DeleteStorageItem(scripthash util.Uint160, key []byte) error
 	return dao.Store.Delete(makeStorageItemKey(scripthash, key))
 }
 
+// StorageItem is a Key-Value pair together with possible const modifier.
+type StorageItem struct {
```

Comment:
`StorageItemAndKey`, `KeyedStorageItem`, whatever, but please not yet another `StorageItem`.

Reply:
Fixed.